src/emu/console/src/main.cpp

```diff
@@ -57,10 +57,17 @@
 #include <imgui.h>
 #include <yaml-cpp/yaml.h>
 
-#if EKA2L1_PLATFORM(WIN32)
-extern "C" _declspec(dllexport) DWORD NvOptimusEnablement = 0x00000001;
+#if defined(_MSC_VER)
+#define EKA2L1_EXPORT __declspec(dllexport)
+#else
+#define EKA2L1_EXPORT __attribute__((dllexport))
 #endif
 
+extern "C" {
+    EKA2L1_EXPORT DWORD NvOptimusEnablement = 0x00000001;
```

Comment:
These are only available on WIN32. Use macro EKA2L1_PLATFORM(WIN32) to explicitly declare them.

Reply:
Nice catch.
I'll try to wrap up the PR tonight if time permits.